HousingRepairsOnlineApi/Gateways/AzureStorageGateway.cs

```diff
@@ -28,5 +29,40 @@ public async Task<string> UploadBlob(string base64Img, string fileExtension)
 
             return blobClient.Uri.AbsoluteUri;
         }
+
+        public async Task<string> GetServiceSasUriForBlob(string fileName, int daysUntilExpiry, string storedPolicyName = null)
```

Comment:
The `fileName` parameter would confuse callers of this method as Azure blob storage doesn't use file names.
Please rename to something more appropriate, i.e. `blobName`
```suggestion
        public async Task<string> GetServiceSasUriForBlob(string blobName, int daysUntilExpiry, string storedPolicyName = null)
```